pupil_src/launchables/player.py

```diff
@@ -387,14 +387,14 @@ def set_window_size():
                            ('System_Timelines', {}),
                            ('Video_Export_Launcher', {}),
                            ('Pupil_From_Recording', {}),
-                           ('Gaze_From_Recording', {})]
+                           ('Gaze_From_Recording', {}),
+                           ('Audio_Playback', {})]
 
-        g_pool.plugins = Plugin_List(g_pool, session_settings.get('loaded_plugins', default_plugins))
+        plugins = session_settings.get('loaded_plugins', default_plugins)
+        g_pool.plugins = Plugin_List(g_pool, plugins)
```

Comment:
Is this change to improve readability or just leftovers from debugging the contents of `plugins`?